src/status_im2/contexts/onboarding/syncing/progress/view.cljs

```diff
@@ -0,0 +1,62 @@
+(ns status-im2.contexts.onboarding.syncing.progress.view
+  (:require [quo2.core :as quo]
+            [utils.i18n :as i18n]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [utils.re-frame :as rf]
+            [react-native.safe-area :as safe-area]
+            [status-im2.contexts.onboarding.syncing.progress.style :as style]
+            [status-im2.contexts.onboarding.common.background.view :as background]))
+
+(defn page-title
+  [pairing-progress?]
+  [rn/view {:style {:padding-horizontal 20}}
+   [quo/text
+    {:accessibility-label :progress-screen-title
+     :weight              :semi-bold
+     :size                :heading-1
+     :style               {:color colors/white}}
+    (if pairing-progress?
+      (i18n/label :t/sync-devices-title)
+      (i18n/label :t/sync-devices-error-title))]
+   [quo/text
+    {:accessibility-label :progress-screen-sub-title
+     :weight              :regular
+     :size                :paragraph-1
+     :style               {:color colors/white}}
+    (if pairing-progress?
+      (i18n/label :t/sync-devices-sub-title)
+      (i18n/label :t/sync-devices-error-sub-title))]])
+
+(defn navigation-bar
+  []
+  [rn/view {:style style/navigation-bar}
+   [quo/page-nav]])
+
+(defn try-again-button
+  [status]
+  (let [profile-color (:color (rf/sub [:onboarding-2/profile]))]
```

Comment:
hmm, would it be better to move the sub to `view` component and just pass this as a param?? 🤔